README.md

```diff
@@ -59,19 +60,21 @@ Output a console log `The sum of 5 and 10 is 15` where the values for 5 and 10 a
 ```js
 const num1 = 5;
 const num2 = 10;
-```
 
+let sum =num1+num2
+console.log("the sum of 5 and 10 is"+sum)
 ### Your solution here:
 5.  How can we make `num3` equal to the sum of `num1` and `num2`?
 ```js
-// your solution here
+const num3=15;
+num3===sum2
 ```
 6.  Use variables `num1`, `num2` and `num3` to fill in the `console.log()` to complete the sentence: 
 
 >The sum of 5 and 10 is 15
 
 ```js
-console.log()
+console.log("the sum of 5 and 10 is"+sum)
```

Comment:
you can answer this question with 

`console.log("the sum of " + num1 + " and " + num2 + " is " num3)`